android/src/main/java/com/voximplant/flutter_voximplant/CallModule.java

```diff
@@ -51,7 +61,9 @@ public class CallModule implements ICallListener, IEndpointListener, EventChanne
         mMessenger = messenger;
         mCall = call;
         mEventChannel = new EventChannel(messenger, "plugins.voximplant.com/call_" + mCall.getCallId());
+        mQualityIssuesEventChannel = new EventChannel(messenger, "plugins.voximplant.com/call_quality_issues");
```

Comment:
we need to separate channels for calls, otherwise if there are multiple calls, we will have several channels with the same name

Reply:
the same for iOS

Reply:
done